src/components/people/LocationButtons.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import LocationButtons, { ButtonFilter } from '@components/people/LocationButtons';
+import { peopleLocationButtonList } from "@globals/constants";
+import { blue, darkerBlue } from '@globals/theme';
+import { ThemeProvider } from "styled-components"
+import theme from '@globals/theme';
```

Comment:
Does ThemeProvider need to be imported here? Also you can probably combine the imports from globals/theme

Reply:
good catch it doesn't anymore! getting rid of ThemeProvider and theme imports